packages/opencensus-core/src/metrics/metrics.ts

```diff
@@ -14,21 +14,29 @@
  * limitations under the License.
  */
 
+import {MetricProducerManager, metricProducerManagerInstance} from './export/metric-producer-manager';
+import {MetricsComponent} from './metric-component';
 import {MetricRegistry} from './metric-registry';
 
+// Class for accessing the default MetricsComponent.
 export class Metrics {
-  private static readonly METRIC_REGISTRY = Metrics.newMetricRegistry();
+  private static readonly METRIC_COMPONENT = new MetricsComponent();
+
+  /**
+   * Returns the global MetricProducerManager.
+   *
+   * @return {MetricProducerManager}.
+   */
+  static getMetricProducerManager(): MetricProducerManager {
```

Comment:
Depending on what `getMetricRegistry` does, you might be able to make these properties instead of getter methods (or even potentially replacing this class for a plain module with consts - just like the previous PR feedback).